supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -129,9 +129,9 @@
 [discrete]
 [[data-warehouse]]
 ==== image:images/yes.png[yes] Data Warehouse (noun)
-*Description*: In Red Hat Virtualization, the Manager includes a comprehensive management history database, which any application can use to extract a range of information at the data center, cluster, and host levels. Installing Data Warehouse creates the `ovirt_engine_history` database, to which the Manager is configured to log information for reporting purposes.
+*Description*: In Red Hat Virtualization, the manager includes a comprehensive management history database, which any application can use to extract a range of information at the data center, cluster, and host levels. Installing Data Warehouse creates the `ovirt_engine_history` database, to which the Manager is configured to log information for reporting purposes.
```

Comment:
I think Manager was correct capitalized as it was (it's capitalized in other RHV entries).

Reply:
OK, I am confused by this one, as _manager_ is usually a generic noun. I feel a more specific name is missing in this section, such as Data Warehouse Manager or Red Hat Virtualization Manager. 


Outside the scope of this PR, so I reverted _Manager_ to its original state.

Reply:
Yeah I think in the context of RHV here, "Manager" is capitalized. See the guidance here: https://redhat-documentation.github.io/supplementary-style-guide/#red-hat-virtualization-manager